src/Mealz/MealBundle/Controller/ParticipantController.php

```diff
@@ -34,7 +68,7 @@ public function delete(Participant $participant): JsonResponse
         $date = $meal->getDateTime()->format('Y-m-d');
         $dish = $meal->getDish()->getSlug();
         $profile = $participant->getProfile()->getUsername();
-        $participant->getCombinedDishes()->clear();
+        $participant->setCombinedDishes(null);
```

Comment:
Why not use an empty combined dishes?

Reply:
It's possible, but a bit more code.

Reply:
Ok, I see the reason, getCombinedDishes returns a copy now.